optuna/integration/__init__.py

```diff
@@ -28,7 +28,6 @@
     "skopt": ["SkoptSampler"],
     "tensorboard": ["TensorBoardCallback"],
     "tensorflow": ["TensorFlowPruningHook"],
-    "tfkeras": ["TFKerasPruningCallback"],
```

Comment:
This line should not be removed, since if we remove this we cannot use `TFKerasPruningCallback` in `optuna` namespace anymore.

Reply:
That makes sense. I reverted the change in https://github.com/optuna/optuna/pull/4662/commits/f8b651d6667205f24ac27788045398196ff6e615.